### ISSUE TITLE
No notification if the save succeeded of failed

### ISSUE THREAD

**Issue by JordyMoos** [JordyMoos/elm-yapm-client#68]
When you add/delete/edit a password, or change your masterkey. Then the library will be saved. But you will not get feedback if the failed succeeded or failed. This should be added.

**Comment by marcusklaas** [comment]
I think it shows an error when it fails?

**Comment by JordyMoos** [comment]
You are right, there is an notification when it fails. Alright thats nice. Shall we still add a notification when it succeeds?
